batch/processor_eth_wst_monitor_txreceipt.py

```diff
@@ -135,19 +149,156 @@ async def finalize_tx(
     """
     Finalize the IbetWST transaction.
     """
-    if wst_tx.tx_type == IbetWSTTxType.DEPLOY:
-        token = (
-            await db_session.scalars(
-                select(Token).where(Token.ibet_wst_tx_id == wst_tx.tx_id).limit(1)
+    match wst_tx.tx_type:
+        case IbetWSTTxType.DEPLOY:
+            token = (
+                await db_session.scalars(
+                    select(Token).where(Token.ibet_wst_tx_id == wst_tx.tx_id).limit(1)
+                )
+            ).first()
+            if token is None:
+                return
+
+            token.ibet_wst_deployed = True
+            token.ibet_wst_address = tx_receipt.get("contractAddress", None)
+            await db_session.merge(token)
+        case IbetWSTTxType.MINT:
+            ibet_wst = IbetWST(wst_tx.ibet_wst_address)
+            event = await EthAsyncContractUtils.get_event_log_by_hash(
+                contract=ibet_wst.contract,
+                event="Mint",
+                block_number=wst_tx.block_number,
+                tx_hash=wst_tx.tx_hash,
+            )
```

Comment:
[nits]
This would be slightly better because it doesn't need an RPC request
```suggestion
            events = ibet_wst.contract.events.Mint().process_receipt(txn_receipt=tx_receipt)
            event = events[0] if len(events) > 0 else None

```

Reply:
Since the proposed approach is less costly, I have adopted it.
https://github.com/BoostryJP/ibet-Prime/pull/831/commits/a83bfbb7e42f97018e046078cda35fc7390b7d10